examples/AWSDriverExample/src/main/java/software/amazon/TelemetryMetricsOTLPExample.java

```diff
@@ -0,0 +1,110 @@
+import io.opentelemetry.exporter.otlp.metrics.OtlpGrpcMetricExporter;
+import io.opentelemetry.exporter.otlp.trace.OtlpGrpcSpanExporter;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.metrics.SdkMeterProvider;
+import io.opentelemetry.sdk.metrics.export.PeriodicMetricReader;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.SimpleSpanProcessor;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.Properties;
+import java.util.concurrent.TimeUnit;
+import software.amazon.jdbc.PropertyDefinition;
+
+public class TelemetryMetricsOTLPExample {
+
+  // User configures connection properties here
+  public static final String POSTGRESQL_CONNECTION_STRING =
+      "jdbc:aws-wrapper:postgresql://atlas-postgres.cluster-czygpppufgy4.us-east-2.rds.amazonaws.com:5432/postgres";
+
+  private static final String USERNAME = "username";
+  private static final String PASSWORD = "password";
+
+  private static final String SQL_DBLIST = "select datname from pg_database;";
+  private static final String SQL_SLEEP = "select pg_sleep(20);";
+  private static final String SQL_TABLELIST = "select * from information_schema.tables where table_schema='public';";
+
+  private final OpenTelemetry openTelemetry;
+  private final OtlpGrpcSpanExporter spanExporter;
+  private final OtlpGrpcMetricExporter metricExporter;
+  private final SdkTracerProvider tracerProvider;
+  private final SdkMeterProvider meterProvider;
+
+  public TelemetryMetricsOTLPExample() {
+    spanExporter = OtlpGrpcSpanExporter.builder().setEndpoint(System.getenv("OTEL_EXPORTER_OTLP_ENDPOINT")).build();
+    metricExporter = OtlpGrpcMetricExporter.builder().setEndpoint(System.getenv("OTEL_EXPORTER_OTLP_ENDPOINT")).build();
+
+    tracerProvider = SdkTracerProvider.builder().addSpanProcessor(SimpleSpanProcessor.create(spanExporter)).build();
+    meterProvider = SdkMeterProvider.builder()
+        .registerMetricReader(PeriodicMetricReader.builder(metricExporter).setInterval(15, TimeUnit.SECONDS).build())
+        .build();
+
+    openTelemetry = OpenTelemetrySdk.builder()
+        .setTracerProvider(tracerProvider)
+        .setMeterProvider(meterProvider)
+        .setPropagators(ContextPropagators.create(W3CTraceContextPropagator.getInstance()))
+        .buildAndRegisterGlobal();
+  }
+
+  public void runExampleQuery(Properties properties) throws SQLException {
+    try (final Connection conn = DriverManager.getConnection(POSTGRESQL_CONNECTION_STRING, properties);
+         final Statement statement = conn.createStatement();
+         final ResultSet rs = statement.executeQuery(SQL_SLEEP)) {
+      System.out.println(Util.getResult(rs));
+    }
+  }
+
+  public static void main(String[] args) throws SQLException {
+    final TelemetryTracingXRayExample example = new TelemetryTracingXRayExample();
```

Comment:
Instantiating an object for a simple demo class is a bit unusual. Can we just make runExampleQueryX() static and use them directly in main()?